targets/frontend/Dockerfile

```diff
@@ -17,7 +17,6 @@ RUN  npx @socialgouv/yarn-workspace-focus-install \
   --cwd targets/frontend \
   --production \
   -- \
-  --cache-folder /dev/shm/yarn \
   --frozen-lockfile \
   --prefer-offline
```

Comment:
We might want to manually erase the cache then ` && yarn cache clean `

Reply:
il faut rajouter ce commit avant de merger ?

Reply:
Ca a l'air ajouté